mathics/builtin/recurrence.py

```diff
@@ -8,11 +8,18 @@
 import sympy
 
 from mathics.builtin.base import Builtin
-from mathics.core.expression import Expression
-from mathics.core.convert import sympy_symbol_prefix, from_sympy
+
+from mathics.core.atoms import IntegerM1
 from mathics.core.attributes import constant
+from mathics.core.convert import sympy_symbol_prefix, from_sympy
+from mathics.core.expression import Expression
+from mathics.core.list import ListExpression
 from mathics.core.symbols import Atom, Symbol
-from mathics.core.systemsymbols import SymbolFunction
+from mathics.core.systemsymbols import SymbolFunction, SymbolRule
+
+
+SymbolPlus = Symbol("Plus")
```

Comment:
SymbolPlus and SymbolTimes are found in atoms

Reply:
Oh, I see. I was pretty sure I had put them in systemsymbols ...

Reply:
atom.py needs them for the Numeric mixin.